python/cuml/preprocessing/__init__.py

```diff
@@ -24,7 +24,7 @@
     MinMaxScaler, MaxAbsScaler, Normalizer, Binarizer, PolynomialFeatures, \
     SimpleImputer, RobustScaler, KBinsDiscretizer
 from cuml._thirdparty.sklearn.preprocessing import scale, minmax_scale, \
```

Comment:
Preprocessing models should remain accessible from `cuml.experimental.preprocessing.` Can you had `maxabs_scale` there?